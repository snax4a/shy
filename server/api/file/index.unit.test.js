/* globals describe, expect, test */
import router from './index';

describe('File API Route Handlers:', () => {
  test('should have unchanged route handlers', () =>
    expect(router.stack).toMatchSnapshot()
  );
});
