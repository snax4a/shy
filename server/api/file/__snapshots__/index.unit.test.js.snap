// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`File API Route Handlers: should have unchanged route handlers 1`] = `
Array [
  Layer {
    "handle": [Function],
    "keys": Array [],
    "name": "bound dispatch",
    "params": undefined,
    "path": undefined,
    "regexp": /\\^\\\\/\\?\\$/i,
    "route": Route {
      "methods": Object {
        "get": true,
      },
      "path": "/",
      "stack": Array [
        Layer {
          "handle": [Function],
          "keys": Array [],
          "method": "get",
          "name": "<anonymous>",
          "params": undefined,
          "path": undefined,
          "regexp": /\\^\\\\/\\?\\$/i,
        },
      ],
    },
  },
  Layer {
    "handle": [Function],
    "keys": Array [
      Object {
        "name": "id",
        "offset": 1,
        "optional": false,
      },
    ],
    "name": "bound dispatch",
    "params": undefined,
    "path": undefined,
    "regexp": /\\^\\\\/\\(\\?:\\(\\[\\^\\\\/\\]\\+\\?\\)\\)\\\\/\\?\\$/i,
    "route": Route {
      "methods": Object {
        "get": true,
      },
      "path": "/:id",
      "stack": Array [
        Layer {
          "handle": [Function],
          "keys": Array [],
          "method": "get",
          "name": "<anonymous>",
          "params": undefined,
          "path": undefined,
          "regexp": /\\^\\\\/\\?\\$/i,
        },
      ],
    },
  },
  Layer {
    "handle": [Function],
    "keys": Array [],
    "name": "bound dispatch",
    "params": undefined,
    "path": undefined,
    "regexp": /\\^\\\\/upload\\\\/\\?\\$/i,
    "route": Route {
      "methods": Object {
        "post": true,
      },
      "path": "/upload",
      "stack": Array [
        Layer {
          "handle": [Function],
          "keys": Array [],
          "method": "post",
          "name": "middleware",
          "params": undefined,
          "path": undefined,
          "regexp": /\\^\\\\/\\?\\$/i,
        },
        Layer {
          "handle": [Function],
          "keys": Array [],
          "method": "post",
          "name": "<anonymous>",
          "params": undefined,
          "path": undefined,
          "regexp": /\\^\\\\/\\?\\$/i,
        },
      ],
    },
  },
  Layer {
    "handle": [Function],
    "keys": Array [
      Object {
        "name": "id",
        "offset": 1,
        "optional": false,
      },
    ],
    "name": "bound dispatch",
    "params": undefined,
    "path": undefined,
    "regexp": /\\^\\\\/\\(\\?:\\(\\[\\^\\\\/\\]\\+\\?\\)\\)\\\\/\\?\\$/i,
    "route": Route {
      "methods": Object {
        "delete": true,
      },
      "path": "/:id",
      "stack": Array [
        Layer {
          "handle": [Function],
          "keys": Array [],
          "method": "delete",
          "name": "middleware",
          "params": undefined,
          "path": undefined,
          "regexp": /\\^\\\\/\\?\\$/i,
        },
        Layer {
          "handle": [Function],
          "keys": Array [],
          "method": "delete",
          "name": "<anonymous>",
          "params": undefined,
          "path": undefined,
          "regexp": /\\^\\\\/\\?\\$/i,
        },
      ],
    },
  },
]
`;
